broker/cloud_run/lsst/lite/main.py

```diff
@@ -0,0 +1,200 @@
+)
+
+app = flask.Flask(__name__)
+
+
+@app.route(ROUTE_RUN, methods=["POST"])
+def run():
+    """Produces a 'lite' alert stream (${survey}-lite). Messages in this stream contain a subset of fields
+    from the original alert packet.
+
+    This module is intended to be deployed as a Cloud Run service. It will operate as an HTTP endpoint
+    triggered by Pub/Sub messages. This function will be called once for every message sent to this route.
+    It should accept the incoming HTTP request and return a response.
+
+    Returns
+    -------
+    response : tuple(str, int)
+        Tuple containing the response body (string) and HTTP status code (int). Flask will convert the
+        tuple into a proper HTTP response. Note that the response is a status message for the web server.
+    """
+    # extract the envelope from the request that triggered the endpoint
+    # this contains a single Pub/Sub message with the alert to be processed
+    envelope = flask.request.get_json()
+    try:
+        alert = pittgoogle.Alert.from_cloud_run(envelope, f"{SURVEY}")
+    except pittgoogle.exceptions.BadRequest as exc:
+        return str(exc), HTTP_400
+
+    TOPIC_LITE.publish(_create_lite_alert(alert), serializer="json")
+
+    return "", HTTP_204
+
+
+def _create_lite_alert(alert: pittgoogle.Alert) -> pittgoogle.Alert:
+    """Creates a "lite" alert containing a subset of the fields of the original alert packet."""
+
+    # create a list of field names that will be included in a lite dictionary
+    source_fields_list = _get_fields(field_name=alert.get_key("source"))
+    object_fields_list = _get_fields(field_name=alert.get_key("object"))
+    ss_object_fields_list = _get_fields(field_name=alert.get_key("ss_object"))
+
+    # create lite dictionaries for each of the fields that will be included in the lite alert
+    object_lite_dict = _create_lite_dict(
+        alert.dict.get(alert.get_key("object")), object_fields_list
+    )
+    prev_sources_lite_dict = _create_prv_sources_lite_dict(
+        alert.dict.get(alert.get_key("prv_sources")), source_fields_list
+    )
+    source_lite_dict = _create_lite_dict(
+        alert.dict.get(alert.get_key("source")), source_fields_list
+    )
+    ssobject_lite_dict = _create_lite_dict(
+        alert.dict.get(alert.get_key("ss_object")), ss_object_fields_list
+    )
+    # create the lite dictionary for the outgoing alert
+    alert_lite_dict = {
+        alert.get_key("alertid"): alert.get("alertid"),
+        alert.get_key("source"): source_lite_dict,
+        alert.get_key("prv_sources"): prev_sources_lite_dict,
+        "prvDiaForcedSources": alert.dict.get("prvDiaForcedSources"),
+        "prvDiaNondetectionLimits": alert.dict.get("prvDiaNondetectionLimits"),
+        alert.get_key("object"): object_lite_dict,
+        alert.get_key("ss_object"): ssobject_lite_dict,
+    }
+
+    return pittgoogle.Alert.from_dict(payload=alert_lite_dict, schema_name=f"{SURVEY}")
```

Comment:
At minimum, pass along the attributes from the incoming alert. Are there other attributes you want to attach here?

Reply:
@troyraen 
> Are there other attributes you want to attach here?

Not at the moment.

> At minimum, pass along the attributes from the incoming alert.

I think that might be repetitive with the current setup. As of right now, the attributes from the incoming alert are the same as `alert_lite` (i.e., if we choose to attach other attributes in the future then the incoming/outgoing alert attributes may differ) and these attributes are attached to outgoing Pub/Sub messages when the `publish` method is executed.

Reply:
There are attributes from the incoming Kafka stream that will be lost if not explicitly attached to this outgoing alert. Try it. I think you need to pass the incoming alert attributes to `Alert.from_dict()` to end up with the desired outcome (all incoming and added attributes preset in outgoing alert). If you do find that there are "added" attributes that don't end up in the outgoing alert, we may just want to de-indent [alert.py line 273](https://github.com/mwvgroup/pittgoogle-client/blob/9e2a4a00245a93d8530ec346773e0994667a648d/pittgoogle/alert.py#L273) so it's called every time `Alert.attributes` is.

Reply:
> There are attributes from the incoming Kafka stream that will be lost if not explicitly attached to this outgoing alert. 

You're absolutely right. Thank you for the reminder. I'll pass them to outgoing messages